vignettes/data-combined-plotting.Rmd

```diff
@@ -0,0 +1,182 @@
+---
+title: "Visualizations with `DataCombined`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Visualizations with `DataCombined`}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  warning = FALSE,
+  message = FALSE,
+  dpi = 300,
+  comment = "#>",
+  out.width = "100%",
+  fig.height = 6,
+  fig.width = 8
+)
+```
+
+## Introduction
+
+You have already seen how the `DataCombined` class can be utilized to store observed and/or simulated data (if not, read [Working with `DataCombined` class](data-combined.html)).
+
+Let's first create a `DataCombined` object, which we will use to demonstrate different visualizations available.
+
+```{r}
+library(ospsuite)
+
+# simulated data
+simFilePath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
+sim <- loadSimulation(simFilePath)
+simResults <- runSimulation(sim)
+outputPath <- "Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)"
+
+# observed data
+obsData <- lapply(
+  c("ObsDataAciclovir_1.pkml", "ObsDataAciclovir_2.pkml", "ObsDataAciclovir_3.pkml"),
+  function(x) loadDataSetFromPKML(system.file("extdata", x, package = "ospsuite"))
+)
+names(obsData) <- lapply(obsData, function(x) x$name)
+
+myDataCombined <- DataCombined$new()
+
+myDataCombined$addSimulationResults(
+  simulationResults = simResults,
+  quantitiesOrPaths = outputPath,
+  groups = "Aciclovir PVB"
+)
+
+myDataCombined$addDataSets(
+  obsData$`Vergin 1995.Iv`,
+  groups = "Aciclovir PVB"
+)
+```
+
+## Time profile plots
+
+Time profile plots visualize measured or simulated values against time and help 
+assess if the observed data (represented by symbols and error bars) match the 
+simulated data (represented by lines).
+
+```{r}
+plotIndividualTimeProfile(myDataCombined)
+```
+
+## Observed versus simulated scatter plot 
+
+Observed versus simulated plots show allow to assess how far simulated results 
+are from observed values.
+
+```{r}
+plotObservedVsSimulated(myDataCombined)
+```
+
+The identity line represents perfect correspondance of simulated values with the 
+observed ones. By default, a "two-fold" are is marked by the dashed lines. 
+The "x-fold" range is defined as values that are `x`-fold above and `1/x`-fold 
+below the observed onces. The user can specify multiple ranges by the `foldDistance` 
+argument.
+
+```{r}
+plotObservedVsSimulated(myDataCombined, foldDistance = c(1.3, 2))
+```
+
+## Residuals versus time or vs simulated scatter plot
+
+Residual plots show if there is a systematic bias in how the simulation represents values either in high-concentration or low-concentration regions, or, alternatively, in early or late time periods.
+
+```{r}
+plotResidualsVsSimulated(myDataCombined)
+```
+
+
+```{r}
+plotResidualsVsTime(myDataCombined)
+```
+
+## Customizing plot aesthetics
```

Comment:
`aesthetics `is a ggplot term. Not sure if all users are familiar with that. Woul replace it with e.g. _configuration_  or _look and feel_